### PR TITLE
chore: add translations for 1.12.0

### PR DESCRIPTION
**Pull request by scottdover**
**Summary**
This updates the translations needed for our next release.  There are two ways to update translations.
 - You can add suggestions for each translated string. [click here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) for more information on adding suggestions.
 - You can update translations using one of the csvs below by editing the file and re-attaching it to this pull request
    - **de**: [de-locale-update-1.12.0.csv](https://github.com/user-attachments/files/17757273/de-locale-update-1.12.0.csv)
    - **es**: [es-locale-update-1.12.0.csv](https://github.com/user-attachments/files/17757274/es-locale-update-1.12.0.csv)
    - **fr**: [fr-locale-update-1.12.0.csv](https://github.com/user-attachments/files/17757275/fr-locale-update-1.12.0.csv)
    - **it**: [it-locale-update-1.12.0.csv](https://github.com/user-attachments/files/17757276/it-locale-update-1.12.0.csv)
    - **ja**: [ja-locale-update-1.12.0.csv](https://github.com/user-attachments/files/17757277/ja-locale-update-1.12.0.csv)
    - **ko**: [ko-locale-update-1.12.0.csv](https://github.com/user-attachments/files/17757278/ko-locale-update-1.12.0.csv)
    - **pt-br**: [pt-br-locale-update-1.12.0.csv](https://github.com/user-attachments/files/17757279/pt-br-locale-update-1.12.0.csv)
    - **zh-cn**: [zh-cn-locale-update-1.12.0.csv](https://github.com/user-attachments/files/17757280/zh-cn-locale-update-1.12.0.csv)

**Testing**
 - [x] Verify **de** strings are translated
 - [x] Verify **es** strings are translated
 - [x] Verify **fr** strings are translated
 - [x] Verify **ja** strings are translated
 - [x] Verify **ko** strings are translated
 - [x] Verify **zh-cn** strings are translated
 - [x] Verify **it** strings are translated
 - [x] Verify **pt-br** strings are translated
 - [x] Verify all strings are captured by running the translation check (https://github.com/sassoftware/vscode-sas-extension/actions/runs/11921714998/job/33226446830)

### CODE REVIEW

**Comment by scottdover** [comment]
There's one translation here that may look a bit confusing: `{basename}_Copy{number}{ext}`. This is used to represent sas notebook files when there is more than one file with a similar name. So we may end up with...
```
- newNotebook.sasnb
- newNotebook_Copy1.sasnb
```

This translation is used to assign file names

File: l10n/bundle.l10n.zh-cn.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   "Connecting to SAS session...": "正在连接到SAS会话...",
   "Connection Type": "连接类型",
   "Converting SAS notebook to flow...": "将 SAS 笔记本转换为流...",
+  "Could not connect to the SAS server.": "Could not connect to the SAS server.",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:18]:
```suggestion
  "Could not connect to the SAS server.": "无法连接到 SAS 服务器。",
```

File: l10n/bundle.l10n.zh-cn.json [hunk 2]
```diff
@@ -31,7 +32,10 @@
   "Enter the SAS compute context.": "输入SAS计算上下文。",
   "Enter the URL": "输入网址",
   "Enter the URL for the SAS Viya server. An example is https://example.sas.com.": "输入SAS Viya服务器的URL。示例为 https://example.sas.com",
+  "Enter the local private key file path": "Enter the local private key file path",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:35]:
```suggestion
  "Enter the local private key file path": "输入本地私钥文件路径",
```

File: l10n/bundle.l10n.zh-cn.json [hunk 3]
```diff
@@ -31,7 +32,10 @@
...
   "Enter the name of the SAS 9 server.": "输入SAS 9 服务器的名称。",
+  "Enter the passphrase for the private key": "Enter the passphrase for the private key",
+  "Enter the password for user: {username}": "Enter the password for user: {username}",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:38]:
```suggestion
  "Enter the passphrase for the private key": "输入私钥的密码",
  "Enter the password for user: {username}": "输入用户 {username} 的密码",
```

File: l10n/bundle.l10n.zh-cn.json [hunk 4]
```diff
@@ -43,7 +47,7 @@
   "Error converting the notebook file to .flw format.": "将笔记本文件转换为 .flw 格式时出错。",
   "Error getting server with ID {id} - {message}": "获取ID为{id} - {message}的服务器时出错",
   "Error getting session with ID {id} - {message}": "获取ID为{id} - {message}的会话时出错",
-  "Failed to connect to Session. Check profile settings.": "无法连接到会话。请检查配置文件设置。",
+  "Error reading private key file: {filePath}, error: {message}": "Error reading private key file: {filePath}, error: {message}",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:50]:
```suggestion
  "Error reading private key file: {filePath}, error: {message}": "读取私钥文件 {filePath} 时出错，错误：{message}",
```

File: l10n/bundle.l10n.zh-cn.json [hunk 5]
```diff
@@ -69,8 +73,11 @@
   "No SAS Connection Profile": "无SAS连接配置文件",
   "No authorization code": "无授权码",
   "Not implemented": "未实现",
+  "Passphrase Required": "Passphrase Required",
+  "Password Required": "Password Required",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:77]:
```suggestion
  "Passphrase Required": "需要密码短语",
  "Password Required": "需要密码",
```

File: l10n/bundle.l10n.zh-cn.json [hunk 6]
```diff
@@ -69,8 +73,11 @@
...
   "Paste authorization code here": "在此处粘贴授权码",
   "Port Number": "端口号",
+  "Private Key File Path (optional)": "Private Key File Path (optional)",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:80]:
```suggestion
  "Private Key File Path (optional)": "私钥文件路径（可选）",
```

File: l10n/bundle.l10n.zh-cn.json [hunk 7]
```diff
@@ -103,6 +109,7 @@
   "Task is complete.": "任务完成。",
   "The SAS session has closed.": "SAS会话已关闭。",
   "The file type is unsupported.": "不支持该文件类型。",
+  "The folder name cannot contain more than 100 characters or have invalid characters.": "The folder name cannot contain more than 100 characters or have invalid characters.",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:112]:
```suggestion
  "The folder name cannot contain more than 100 characters or have invalid characters.": "文件夹名称不能超过 100 个字符或包含无效字符。",
```

File: l10n/bundle.l10n.zh-cn.json [hunk 8]
```diff
@@ -111,6 +118,7 @@
   "The {selected} SAS connection profile has been deleted from the settings.json file.": "已从settings.json文件中删除{selected} SAS连接配置文件。",
   "There was an error executing the SAS Program. See [console log](command:workbench.action.toggleDevTools) for more details.": "SAS程序执行出错。请参见[开发控制台日志](command:workbench.action.toggleDevTools)获取更多信息。",
   "This platform does not support this connection type.": "该平台不支持该连接类型。",
+  "To use the SSH Agent or a password, leave blank.": "To use the SSH Agent or a password, leave blank.",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:121]:
```suggestion
  "To use the SSH Agent or a password, leave blank.": "要使用 SSH 代理或密码，请留空。",
```

File: l10n/bundle.l10n.zh-cn.json [hunk 9]
```diff
@@ -127,7 +135,9 @@
   "Unable to restore folder.": "无法恢复文件夹。",
   "Unable to upload files.": "无法上传文件。",
   "Uploading files...": "正在上传文件...",
+  "User {username} {prompt}": "User {username} {prompt}",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:138]:
```suggestion
  "User {username} {prompt}": "用户 {username} {prompt}",
```

File: l10n/bundle.l10n.zh-cn.json [hunk 10]
```diff
@@ -127,7 +135,9 @@
...
   "View SAS Table": "查看 SAS 表",
   "You can also specify connection profile using the settings.json file.": "您还可以使用settings.json文件指定连接配置文件。",
-  "You must save your file before you can rename it.": "您必须先保存文件，然后才能重命名它。"
+  "You must save your file before you can rename it.": "您必须先保存文件，然后才能重命名它。",
+  "{basename}_Copy{number}{ext}": "{basename}_Copy{number}{ext}"
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:142]:
```suggestion
  "{basename}_Copy{number}{ext}": "{basename}_副本{number}{ext}"
```

File: package.nls.zh-cn.json [hunk 11]
```diff
@@ -49,6 +49,7 @@
   "configuration.SAS.connectionProfiles.profiles.sasOptions": "SAS 连接 SAS 选项",
   "configuration.SAS.connectionProfiles.profiles.ssh.host": "SAS SSH 连接 SSH 主机",
   "configuration.SAS.connectionProfiles.profiles.ssh.port": "SAS SSH 连接端口",
+  "configuration.SAS.connectionProfiles.profiles.ssh.privateKeyFilePath": "SAS SSH Connection private key file path",
```
Comment (scnwwu) [inline package.nls.zh-cn.json:52]:
```suggestion
  "configuration.SAS.connectionProfiles.profiles.ssh.privateKeyFilePath": "SAS SSH 连接私钥文件路径",
```

File: package.nls.zh-cn.json [hunk 12]
```diff
@@ -71,7 +72,9 @@
   "themes.SAS.dark": "SAS 深色",
   "themes.SAS.highContrast": "SAS 高对比度",
   "themes.SAS.light": "SAS 浅色",
+  "views.SAS.explorer": "SAS Content",
```
Comment (scnwwu) [inline package.nls.zh-cn.json:75]:
```suggestion
  "views.SAS.explorer": "SAS 内容",
```

File: package.nls.zh-cn.json [hunk 13]
```diff
@@ -71,7 +72,9 @@
...
   "views.SAS.libraries": "库",
+  "views.SAS.serverExplorer": "SAS Server",
```
Comment (scnwwu) [inline package.nls.zh-cn.json:77]:
```suggestion
  "views.SAS.serverExplorer": "SAS 服务器",
```

**Comment by meilanji** [comment]
[ko-locale-update-1.12.0.csv](https://github.com/user-attachments/files/17772187/ko-locale-update-1.12.0.csv)
Korean translation.

**Comment by vrejany** [comment]
[fr-locale-update-1.12.0.csv](https://github.com/user-attachments/files/17778114/fr-locale-update-1.12.0.csv)

**Review by scnwwu — APPROVED** [review]

**Comment by Criptic** [comment]
[de-locale-update-1.12.0.csv](https://github.com/user-attachments/files/17867556/de-locale-update-1.12.0.csv)

**Pull request merged**